backend/pkg/api/data_access/data_access.go

```diff
@@ -253,6 +253,9 @@ func createDataAccessService(cfg *types.Config) *DataAccessService {
 func (d *DataAccessService) StartDataAccessServices() {
 	// Create the services
 	d.services = services.NewServices(d.readerDb, d.writerDb, d.alloyReader, d.alloyWriter, d.clickhouseReader, d.bigtable, d.persistentRedisDbClient)
+
+	// Initialize repositories
+	d.initNotifications()
```

Comment:
I would prefer if you could call the method `registerNotificationInterfaceTypes` otherwise one might confuse that the call initialized the full notification system